src/HttpClient.php

```diff
@@ -0,0 +1,59 @@
+<?php namespace DocRaptor;
+
+use DocRaptor\Exception\BadRequestException;
+use DocRaptor\Exception\ForbiddenException;
+use DocRaptor\Exception\UnauthorizedException;
+use DocRaptor\Exception\UnexpectedValueException;
+use DocRaptor\Exception\UnprocessableEntityException;
+use Exception;
+
+/**
+ * Class HttpClient
+ * @package DocRaptor
+ */
+class HttpClient implements HttpTransferInterface
+{
+    /**
+     * @param string $uri
+     * @param array $postFields
+     * @return mixed
+     * @throws Exception
+     */
+    public function doPost($uri, array $postFields)
```

Comment:
I think this method needs to be static or the way the class is instantiated should be slightly altered.

It seems like expecting a user of a library to know its internals to use it is a little too far. For example, to make a request, I know have to do something like:
```php
use DocRaptor/ApiWrapper;
use DocRaptor/HttpClient;
apiWrapper = new ApiWrapper(new HttpClient(), 'some_key');
```
I don't think having that call inside my constructor is gonna be very valuable. I would propose changing the `ApiWrapper` constructor

Now if we changed the params to the `ApiWrapper` constructor to look like (warning: possibly invalid php incoming):
```php
public function __construct($api_key = null, HttpTransferInterface $httpClient = new HttpClient())
```
*or*
```php
public function __construct($api_key = null, $httpClientClass = HttpClient)
```
and then just construct whatever type of object is passed in (assuming it adheres to the interface), I think that works out better.

Alternately, could just allow it to be null and construct the client inside the wrapper constructor.